scripts/cloudfront_routing.js

```
@@ -0,0 +1,36 @@
+      // Add other routes that need special handling:
+      // e.g., old paths redirected to new paths or default documents for sections.
+      // Example:
+      // "/old-page": "/new-page",      // redirect '/old-page' -> '/new-page'
+      // Internationalization (i18n) examples for language-specific homepages:
+      "/en": "/en/index.html",          // '/en' -> '/en/index.html' (English homepage)
+      "/en/": "/en/index.html",         // '/en/' -> '/en/index.html'
+      // You can add "/fr": "/fr/index.html", "/fr/": "/fr/index.html", etc., for other languages.
+    };
+  
+    // If the request URI matches a key in routeMap, rewrite it
+    if (routeMap.hasOwnProperty(uri)) {
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)